master/reference/calicoctl/resources/profile.md

```diff
@@ -59,14 +59,19 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| action      | Action to perform when matching this rule. | allow, deny, log | string | | 
+| action      | Action to perform when matching this rule. | allow, deny, log, pass | string | | 
 | protocol    | Positive protocol match.  | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | notProtocol | Negative protocol match. | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | icmp        | ICMP match criteria.     | | [ICMP](#icmp) | |
 | notICMP     | Negative match on ICMP. | | [ICMP](#icmp) | |
 | source      | Source match parameters. |  | [EntityRule](#entityrule) | |
 | destination | Destination match parameters. |  | [EntityRule](#entityrule) | |
 
+An `action` of `pass` will skip over the remaining Policies and jump to the
```

Comment:
I think Policies is not correct (it suggests a Policy resource, but we are now in the context of the profile).  Perhaps one of:

-  remaining policies
-  remaining rules in the Profile
-  remaining match rules in the Profile

or some variation.  WDYT?